src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -630,6 +646,10 @@ public String getDisplayName() {
             return Hudson.getInstance().<AbstractBuildParameterFactory,
               Descriptor<AbstractBuildParameterFactory>>getDescriptorList(AbstractBuildParameterFactory.class);
         }
+        
+        public boolean isItemGroup(AbstractProject project){
```

Comment:
Better to Restrict this method by `@Restricted(DoNotUse.class)` since it is only for UI